src/univariate/continuous/normalcanon.jl

```diff
@@ -1,7 +1,9 @@
 """
     NormalCanon(η, λ)
 
-Canonical Form of Normal distribution
+The canonical form of the normal distribution, which is an [exponential family distribution](http://en.wikipedia.org/wiki/Exponential_family).
+The two *canonical parameters* are ``\\eta = \\sigma^{-1} \\mu`` and ``\\lambda = \\sigma^{-1}``,
+where ``\\mu`` is the mean and ``\\sigma`` is the standard deviation.
```

Comment:
Minor suggestion:
```suggestion
Normal distribution with canonical parameters `η` and `λ`.

The two *canonical parameters* of a normal distribution ``\\mathcal{N}(\\mu, \\sigma^2)`` with mean ``\\mu`` and
standard deviation ``\\sigma`` are ``\\eta = \\sigma^{-1} \\mu`` and ``\\lambda = \\sigma^{-1}``.
```

Reply:
Sure, I just followed the `MvNormalCanon` docstring.
You made a mistake in your suggestion though,
it's eta = sigma^(-2) * mu and lambda = sigma^(-2).

Feel free to commit your suggestion once you've fixed it:)

Reply:
Oh! Hahah I just saw I made the same mistake in my patch :see_no_evil: 

Here's the fixed version:
```suggestion
Canonical parametrisation of the Normal distribution with canonical parameters `η` and `λ`.

The two *canonical parameters* of a normal distribution ``\\mathcal{N}(\\mu, \\sigma^2)`` with mean ``\\mu`` and
standard deviation ``\\sigma`` are ``\\eta = \\sigma^{-2} \\mu`` and ``\\lambda = \\sigma^{-2}``.
```